zuul.d/run.yaml

```diff
@@ -22,3 +31,11 @@
       command: "ansible-playbook -i inventories/uc-engine test.yml"
       args:
         chdir: "{{ zuul.project.src_dir }}"
+
+    - name: Get service facts
+      become: yes
+      service_facts:
```

Comment:
the empty key is needed?

Reply:
yes that's the mandatory name of the task